packages/test-kit/src/with-feature.ts

```diff
@@ -205,19 +205,17 @@ export function withFeature(withFeatureOptions: IWithFeatureOptions = {}) {
 
             const browserContext = await browser.newContext(browserContextOptions);
             browserContexts.add(browserContext);
-            const featurePage = await browserContext.newPage();
-            trackPage(featurePage);
-
-            function trackPage(page: playwright.Page) {
-                hookPageConsole(page, isNonReactDevMessage);
+            browserContext.on('page', (page) => {
```

Comment:
no cleanup upon disposal?

Reply:
We're closing the browser context, which closes the pages of that context.